src/store/create-conversation/saga.ts

```diff
@@ -56,7 +56,6 @@ export function* createConversation(action) {
 
 export function* saga() {
   yield fork(authWatcher);
-  yield fork(handleCreateConversation);
```

Comment:
You can see below we're already listening for this event. No reason to listen here.